public_html/events.php

```diff
@@ -346,8 +346,8 @@ function print_events($events, $is_past_event) {
 
 $title = 'Events';
 $subtitle = 'Details of past and future nf-core meetups.';
-$md_github_url = 'https://github.com/sanger-tol/nf-co.re/tree/main/markdown/events';
-$header_btn_url = 'https://nf-co.re/events/rss';
+$md_github_url = 'https://github.com/sanger-tol/pipelines-website/tree/main/markdown/events';
+$header_btn_url = 'https://pipelines.tol.sanger.ac.uk/events/rss';
```

Comment:
I think this should work because it's only used in the HTML
```suggestion
$header_btn_url = '/events/rss';
```

Reply:
It works. But probably we should use the live rss url everywhere.

nf-core has removed RSS events on their new site. We probably just remove this as well, at least this button.